german/Hilfeschrei/Vertrauen/insMaul/gloeckchen/klingeln/kreuzfahrt/kreuzfahrt.md

```diff
@@ -0,0 +1,3 @@
+ ... es dich hinaus katapultiert aus dem Wal; wie ein Torpedo schießt du durch das salzige Wasser, durchbrichst die Oberfläche und landest mit einem königlichen Bauchplatscher in einem chlorhaltigen Pool. Mit letzter Ktaft hebst du den Kopf und während du noch nach Luft ringst und Wasser spuckst, merkst du : Oh nein, ich bin auf einem Kreuzfahrtschiff gelandet! 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.